packages/config/__tests__/ConfigBuilder.test.ts

```diff
@@ -245,4 +245,116 @@ describe("Config Builder tests", () => {
             expect(builtConfig).toEqual(expectedConfig);
         });
     });
+
+    describe("convert", () => {
+        const mockSecureLoad = jest.fn().mockReturnValue("\"area51\"");
+
+        beforeAll(() => {
+            jest.spyOn(CredentialManagerFactory, "manager", "get").mockReturnValue({
+                load: mockSecureLoad
+            } as any);
+        });
+
+        it("should successfully convert multiple v1 profiles to config object", async () => {
+            jest.spyOn(ProfileIO, "getAllProfileDirectories").mockReturnValueOnce(["fruit", "nut"]);
+            jest.spyOn(ProfileIO, "getAllProfileNames")
+                .mockReturnValueOnce(["apple", "banana", "coconut"])
+                .mockReturnValueOnce(["almond", "brazil", "cashew"]);
+            jest.spyOn(ProfileIO, "readMetaFile")
+                .mockReturnValueOnce({ defaultProfile: "apple" } as any)
+                .mockReturnValueOnce({ defaultProfile: "brazil" } as any);
+            jest.spyOn(ProfileIO, "readProfileFile")
+                .mockReturnValueOnce({ color: "green", secret: "managed by A" })
+                .mockReturnValueOnce({ color: "yellow", secret: "managed by B" })
+                .mockReturnValueOnce({ color: "brown", secret: "managed by C" })
```

Comment:
I'm not asking for this to be done in this PR.
It would be nice to be able to integration/system -test multiple managers and how the migration works for all of them.
Maybe when we get to formalize the overrides work ? 😋